README.md

```diff
@@ -1,83 +1,75 @@
 # dandiarchive [![CircleCI](https://circleci.com/gh/dandi/dandiarchive/tree/master.svg?style=svg)](https://circleci.com/gh/dandi/dandiarchive/tree/master)
-Infrastructure and code for the dandiarchive
+Infrastructure and code for the DANDI Archive.
 
 Folders in this repo:
 
-- Ansible: Ansible code for deploying the girder.dandiarchive.org site.
-- web: Vue.js code for holding a Girder Web Components based client deployed at gui.dandiarchive.org .
-- girder-dandi-archive: A Girder plugin for customizing the DANDI archive Girder instance.
+- `girder-dandi-archive`: A Girder plugin which serves the DANDI Archive.
```

Comment:
I am not quite sure what "serves" means here.  "customizing ... Girder instance" was somewhat more accessible to me

Reply:
Maybe "A Girder plugin which adds the custom endpoints for the DANDI Archive."

Reply:
Sounds good to me.